serverless/functions/Authorizer.yml

```diff
@@ -0,0 +1,5 @@
+iconikAuthorizer:

```

Comment:
i suggest to use prefix like `authorizer` to known that this lambda is an auhtorizer(by name in logs or somewhere else).

Reply:
Yes, it has `Auhtorizer at the and`, but it`s good to have at the beginning for filtering purpose.

Reply:
@khoroshilov-flo I like the idea with prefixes. It can really help with finding a proper Lambda function in AWS Console. How about adding the information about prefixes like `api`, `schedule`, `authorizer` to the regulations?

Reply:
I don`t mind